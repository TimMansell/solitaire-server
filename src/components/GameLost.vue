<template>
  <GameOverlay center-content show-logo data-test="game-lost">
    <template #title> Sorry, no more Moves! </template>
    <template #msg>
      <GameSummary />
    </template>
    <template #buttons>
      <NewGameButton />
      <ShowBoardButton />
    </template>
  </GameOverlay>
</template>

<script>
import GameOverlay from '@/components/GameOverlay.vue';
import GameSummary from '@/components/GameSummary.vue';
import NewGameButton from '@/components/NewGameButton.vue';
import ShowBoardButton from './ShowBoardButton.vue';

export default {
  name: 'GameLost',
  components: {
    GameOverlay,
    NewGameButton,
    GameSummary,
    ShowBoardButton,
  },
};
</script>
