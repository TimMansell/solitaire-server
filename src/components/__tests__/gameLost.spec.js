import { shallowMount } from '@vue/test-utils';
import GameLost from '@/components/GameLost.vue';

describe('GameLost.vue', () => {
  it('matches snapshot', () => {
    const wrapper = shallowMount(GameLost);

    expect(wrapper).toMatchSnapshot();
  });
});
