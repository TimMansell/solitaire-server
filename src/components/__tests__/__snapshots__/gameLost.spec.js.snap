// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GameLost.vue matches snapshot 1`] = `
<gameoverlay-stub centercontent="true" showlogo="true" data-test="game-lost">
  Sorry, no more Moves!
  <gamesummary-stub></gamesummary-stub>
  <newgamebutton-stub></newgamebutton-stub>
  <showboardbutton-stub></showboardbutton-stub>
</gameoverlay-stub>
`;
